lib/modules/manager/nuget/update.ts

```diff
@@ -30,8 +31,23 @@ export function bumpPackageVersion(
 
   try {
     const project = new XmlDocument(content);
-    const versionNode = project.descendantWithPath('PropertyGroup.Version')!;
+    const versionNode = findVersion(project);
+    if (!versionNode) {
+      logger.warn(
```

Comment:
should we really warn here? 🤔

Reply:
Yeah, I dunno. It feels a little like the warning on falsy `currentValue`, but I confess I don't really understand what that really is, so I don't know if the comparison is warranted. (It feels a little strange to both be passed in a version and extract it as well and then require the two match; and only one of the `bumpPackageVersion()` implementations actually uses `currentValue` in any meaningful way.)

Since a project file doesn't require a version to be set, it's not a warning about the validity of that file, but it does seem to me that since `bumpVersion` has to be explicitly enabled, it would make sense that when this isn't possible, the user should be alerted. Perhaps this only rises to the level of `INFO`, though?